deltachat-ios/Controller/QrPageController.swift

```diff
@@ -12,10 +12,18 @@ class QrPageController: UIPageViewController, ProgressAlertHandler {
     private lazy var qrCodeHint: String = {
         var qrCodeHint = ""
         if dcContext.isConfigured() {
-            let contact = DcContact(id: Int(DC_CONTACT_ID_SELF))
+            // we cannot use dc_contact_get_displayname() as this would result in "Me" instead of the real name
+            let name = dcContext.getConfig("displayname") ?? ""
+            let addr = dcContext.getConfig("addr") ?? ""
+            var nameAndAddress = ""
+            if name.isEmpty {
+                nameAndAddress = addr
+            } else {
+                nameAndAddress = name + " (" + addr + ")"
```

Comment:
you could use a imo little bit 'swiftier' syntax: 
`nameAndAddress = "\(name) (\(addr))"`

Reply:
When changing the profile name the qrCodeHint does not get updated. We should remove `lazy` and make this a computed property.

Reply:
(the syntax highlighting in xcode makes the parenthesis around addr better readable)